<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QReagentScanDialog</class>
 <widget class="BaseDlg" name="QReagentScanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QReagentScanDialog</string>
  </property>
  <widget class="QWidget" name="widgetDevices" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>616</width>
     <height>60</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="BottomFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>720</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QRadioButton" name="scan_selected_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>541</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描指定试剂位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>341</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>例如： 2，4，6-10</string>
    </property>
    <property name="colors" stdset="0">
     <string>disabled</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="select_all_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>521</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>扫描全部试剂位</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="select_edit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="BottomBtnFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>633</width>
      <height>52</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>633</width>
      <height>52</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>633</width>
      <height>52</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
     <property name="spacing">
      <number>14</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>366</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
       <property name="buttons" stdset="0">
        <string>btn_dlg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="minimumSize">
        <size>
         <width>116</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
       <property name="buttons" stdset="0">
        <string>btn_dlg</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_margin_detect" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>671</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QCheckBox" name="allowance_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>511</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>余量检测</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_fast">
        <property name="text">
         <string>快速余量检测</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_sync">
        <property name="text">
         <string>同步余量检测</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>BottomBtnFrame</zorder>
   <zorder>scan_selected_btn</zorder>
   <zorder>label</zorder>
   <zorder>select_all_btn</zorder>
   <zorder>select_edit</zorder>
   <zorder>widget_margin_detect</zorder>
  </widget>
  <zorder>BottomFrame</zorder>
  <zorder>widgetDevices</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
